students/arkadiusz_kasprzyk/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,32 @@
+'''
+title: apple_sharing
+author: arkadiusz.kasprzyk@tieto.com
+date: 2018-03-05
```

Comment:
If the files were elements of one project then OK. But they are not - each is separate being.
I'd say that DRY principle is about code not about metainformations in comments or headings.
Of curse, it's rarely good to repeat yourself but sometimes it's vary useful; in case of headings - for me, it's harmless if only not too much of it. E.g. repeating title of a file allows to change the name of the file without worry that info about its content is lost forever. And so on.
The term "redundancy" is not always pejorative: information replicated is protected against vanishing. 
Obviously, it doesn't work in case of code, where RY makes it prone to bugs and less readable. No doubt about it.

Reply:
The redundancy is not pejorative, when computer does it ;)